.github/CODEOWNERS

```diff
@@ -341,7 +341,7 @@
 # ServiceOwners:                                     @kushagraThapar @TheovanKraay @pjohari-ms
 
 # PRLabel: %Cosmos
-/sdk/cosmos/                                         @kushagraThapar @FabianMeiswinkel @kirankumarkolli @xinlian12 @milismsft @aayush3011 @simorenoh @jeet1995 @Pilchie @nehrao1
+/sdk/cosmos/                                         @kirankumarkolli @azure-cosmos-java-sdk-connectors
```

Comment:
@kushagraThapar I think it needs to be:

@Azure/azure-cosmos-java-sdk-connectors

Reply:
yes, update this to use `@Azure/azure-cosmos-java-sdk-connectors `

Reply:
Updated, thanks :)